src/ConfigProvider.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Answear\EcontBundle;
+
+class ConfigProvider
+{
+    private const URL = 'http://ee.econt.com/';
+    private const SERVICE_URL = '/services/Nomenclatures/';
+
+    public function getUrl(): string
+    {
+        return static::URL;
+    }
+
+    public function getServiceUrl(): string
```

Comment:
The method name suggests some url is returned, in fact it is just a plain string "/services/Nomenclatures". This doesn't fit well with the method above

Reply:
Maybe getServiceURI will be better.